api/src/tasks/import-task.js

```diff
@@ -93,18 +115,10 @@ function getDataSourceConfig(dataSourceType) {
     return null;
   }
 
-  switch (dataSourceType) {
-    case 'epic':
-      return {
-        dataSourceLabel: 'epic',
-        dataSourceClass: require('../integrations/epic/datasource')
-      };
-    case 'nris-epd':
-      return {
-        dataSourceLabel: 'nris-epd',
-        dataSourceClass: require('../integrations/nris/datasource')
-      };
-    default:
-      return null;
-  }
+  // dataSourceType will match the name of a directory for the given
+  // integration in /src/integrations/<dataSourceType>/
+  return {
+    dataSourceLabel: dataSourceType,
+    dataSourceClass: require(`../integrations/${dataSourceType}/datasource`)
```

Comment:
@marklise I don't think this works for `nris` currently, as the folder structure is `integrations/nris/stuff` but all of our calls, and swagger, use 'nris-epd' as the dataSourceType.